HISTORY.rst

```diff
@@ -6,6 +6,17 @@ Release History
 unreleased
 +++++++++++++++
 
+**IoT Hub Update**
```

Comment:
Lets put these notes under 0.18.1 since that is going out next, also tiny nit what about "**IoT Hub updates**" for the header?